docs/setting-up/server/docker.md

```diff
@@ -73,6 +82,84 @@ How to run PMM Server with Docker based on our [Docker image].
 
 5. Visit `https://localhost:443` to see the PMM user interface in a web browser. (If you are accessing the docker host remotely, replace `localhost` with the IP or server name of the host.)
 
+### Run Docker with the host folder
+
+1. Pull the image.
+
+    ```sh
+    docker pull percona/pmm-server:2
+    ```
+
+
+2. Backup the data.
+
+    ```sh
+    docker cp <containerId>:/file/path/within/container /host/path/target
+    ```
+
+3. Run the image.
+
+    ```sh
+    docker run -v $HOME/srv:/srv -d --restart always --publish 80:80 --publish 443:443 --name pmm-server perconalab/pmm-server-fb:PR-2534-517015b
```

Comment:
Instead HOME I suggest using: $DATA_DIR:
```
export DATA_DIR=$HOME/srv
docker run -v $DATA_DIR:/srv -d --restart always --publish 80:80 --publish 443:443 --name pmm-server perconalab/pmm-server-fb:PR-2534-517015b
```
where DATA_DIR is a directory where you want to store the state for PMM.